test/org/sagebionetworks/bridge/dynamodb/DynamoUploadDaoTest.java

```diff
@@ -230,6 +230,7 @@ public void canPaginateUploads() throws Exception {
         List<? extends Upload> items = paginatedList.getItems();
         assertEquals(1, items.size()); // only return 1 upload in each page
         assertEquals(1, paginatedList.getPageSize());
+        assertEquals(1, paginatedList.getTotal());
```

Comment:
I think we decided to remove getTotal() from the test, since it wasn't a meaningful value. (Here and two other places in this file.)